all_protos/injective/evm/v1/transaction_logs.proto

```
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package injective.evm.v1;
+
+import "injective/evm/v1/log.proto";
```

Buf (1.54.0)

import "injective/evm/v1/log.proto": file does not exist

(COMPILE)